packages/help/Help.js

```diff
@@ -111,13 +111,23 @@ export const triggerFieldHelp = (id) => {
   });
 };
 
+const handleKeyPress = (e, id) => {
+  if (e.key === 'Enter') {
+    triggerFieldHelp(id);
+  }
+};
+
 export const FieldHelpIcon = ({ color, size, id }) => (
   <Icon
+    role="link"
     data-testid="field-help-icon"
     name="help-circle"
     size={size}
     color={color}
     onClick={() => triggerFieldHelp(id)}
+    tabIndex={0}
+    onKeyPress={(e) => handleKeyPress(e, id)}
+    aria-label="This link opens the help portal, press the enter key or click the icon to open"
```

Comment:
This is probably too verbose which will be frustrating for users tabbing through when there's multiple help icons on a form.

The recommendation from level access was to just leave the aria-label as "help" but make the aria-describedby the id of the corresponding label element so users can differentiate different helps icons on a page.

Reply:
@LauRoxx - sounds good! I'll change the PR. I'll try the aria-describedby route. thanks!

Reply:
@LauRoxx - just updated the PR